common/repositories/base/base_account_repository.h

```diff
@@ -150,6 +159,9 @@ class BaseAccountRepository {
 		entry.ls_id          = "eqemu";
 		entry.lsaccount_id   = 0;
 		entry.gmspeed        = 0;
+		entry.invulnerable   = 0;
+		entry.flymode        = 0;
```

Comment:
Can use `GravityBehavior::Ground` here maybe?

Reply:
Base repositories are code generated and are fairly dumb. This is to just represent "zero" or "default" values and wouldn't yield much improvement to readability here to try and inject constants